engine/lib/generic_printer/generic_printer.ml

```diff
@@ -231,14 +231,15 @@ module Make (F : Features.T) = struct
       [local] is true, otherwise it prints the full path, separated by
       `module_path_separator`. *)
 
-      method quote (quote : quote) : document =
+      method quote ~contents ~witness:_ : document =
```

Comment:
why not take an argument of type `quote` here, which has both the contents and witness?

Reply:
My understanding is that the method signature is generated from the AST so we don't get to choose. But maybe I am wrong and there is a better way. @W95Psp will know.